src/charts/helpers/optimizedResize.js

```diff
@@ -48,6 +48,24 @@ const optimizedResize = (function() {
         }
     };
 
+    const resizeMainHorizontal = () => {
+        let newWidth = document.querySelector('main').offsetWidth;
```

Comment:
I'm not familiar enough with the codebase to know what `main` is. Is that something that's always guaranteed to be present?

Reply:
That's the element of the 'main' section of the documentation.
It is not safe that it will always be there, and I cannot find how to add a class there to make it easier.
adding a comment.